contrib/podmanimage/upstream/Dockerfile

```diff
@@ -17,7 +17,7 @@ ENV GOPATH=/root/podman
 # to the container.
 # Finally remove the podman directory and a few other packages
 # that are needed for building but not running Podman
-RUN yum -y update; rpm --restore --quiet shadow-utils; yum -y install --exclude container-selinux \
+RUN dnf -y update; rpm --restore shadow-utils 2>/dev/null;  yum -y install --exclude container-selinux \
```

Comment:
Is the change from `yum` to `dnf` here intentional?  The other call on this line, and in the ones in the other Dockerfiles, aren't changed.

Reply:
No this is just a cut and paste error, although not sure why these are inconsistent.  I will leave it as yum.